packages/client/src/views/SysAdmin/Config/Integrations/Integrations.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import { Content } from '@opencrvs/components/lib/Content'
+import { useIntl } from 'react-intl'
+import { messages } from '@client/i18n/messages/views/formConfig'
+import { SecondaryButton } from '@client/../../components/lib/buttons'
+import {
+  ListViewItemSimplified,
+  ListViewSimplified
+} from '@opencrvs/components/lib/ListViewSimplified'
+import { Frame } from '@opencrvs/components/lib/Frame'
+import { Navigation } from '@client/components/interface/Navigation'
+import { Header } from '@client/components/Header/Header'
+import { Plus, VerticalThreeDots } from '@client/../../components/lib/icons'
+import { Pill, ToggleMenu } from '@client/../../components/lib'
+import { constantsMessages } from '@client/i18n/messages'
+
+interface IIntegration {
+  name: string
+  status: string
+  systemId: string
+}
+
+export const statuses = {
+  PENDING: 'pending',
+  ACTIVE: 'active',
+  DISABLED: 'disabled',
+  DEACTIVATED: 'deactivated'
+}
+
+export function IntegrationList() {
+  const intl = useIntl()
+
+  return (
+    <Frame
+      header={<Header />}
+      navigation={<Navigation loadWorkqueueStatuses={false} />}
+      skipToContentText={intl.formatMessage(
+        constantsMessages.skipToMainContent
+      )}
+    >
+      <Content
+        title={intl.formatMessage(messages.integrations)}
+        topActionButtons={[
+          <SecondaryButton
+            key="1"
+            id="myButton"
+            onClick={function noRefCheck() {}}
+          >
+            <Plus /> Create Client
+          </SecondaryButton>
```

Comment:
hi see Button with icon from here: https://ui-kit.farajaland-staging.opencrvs.org/?path=/docs/controls-button--button

```ts
<Button type="secondary">
  <Plus /> Create client
</Button>
```

Also I think "Create client" would need to be added to translations. I think we should add `packages/client/src/i18n/messages/views/integrations.ts` with:

```
import { defineMessages } from 'react-intl'

const messagesToDefine = {
  createClient: {
    id: 'integrations.createClient',
    defaultMessage: 'Create client',
    description: 'Label for the button creating client'
  }
}

export const integrationMessages = defineMessages(messagesToDefine)
```
then you can import and use `intl.formatMessage(integrationMessages.createClient)` here.